strings/rust/palindrome/README.md

```diff
@@ -0,0 +1,35 @@
+
+`palindrome`
+
+Language: Rust ( `rust` )
+
+Main/Entry File: [`src/main.rs`](src/main.rs)
+
+Algorithm File: [`src/main.rs`](src/main.rs)
+
+Author: [Goutham Krishna](https://github.com/gauthamkrishna9991/)
+
+---
+
```

Comment:
Hi @gauthamkrishna9991,
Please delete theses lines, you not need them, due to we are using git

Reply:
Done.